oca_dependencies.txt

```diff
@@ -4,3 +4,4 @@ server-tools
 account-fiscal-rule
 reporting-engine
 bank-payment
+account-payment
```

Comment:
I can't see account_payment_order module in oca_dependencies.txt this module is not in the repositories OCA/bank-payment 8.0 and OCA/account-payment 8.0

Reply:
The module is in repository bank-payment version 10.0 https://github.com/OCA/bank-payment/tree/10.0/account_payment_order

Reply:
Ok @mbcosta sorry for my mistake.